packages/env/lib/env.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const fs = require( 'fs' );
+const dockerCompose = require( 'docker-compose' );
+const NodeGit = require( 'nodegit' );
+const wait = require( 'util' ).promisify( setTimeout );
+
+/**
+ * Internal dependencies
+ */
+const createDockerComposeConfig = require( './create-docker-compose-config' );
+
+// Config Variables
+const pluginPath = process.cwd();
+const pluginName = path.basename( pluginPath );
+const pluginTestsPath = fs.existsSync( './packages' ) ? '/packages' : '';
+const dockerComposeOptions = {
+	config: path.join( __dirname, 'docker-compose.yml' ),
```

Comment:
Specifying the path to `docker-compose.yml` file causes Docker to not load any `docker-composer.override.yml` file which may exist.

The override file is often useful for overriding default behaviour, it would likely be useful to continue supporting it.

Reply:
Docker is running inside the package folder so it wouldn't use the override anyway, but what is a valid use case for this? I'm concerned that it will cause lots of headaches for people that already use docker in their repo for other things and have docker files lying around.

We could introduce an option that takes a path to a docker compose file to merge in, but I think that's not how a tool like this should work.

If the use case the override enables is common enough that a lot of people request it, then it should be something the CLI infers as a need and supports out of the box. If it's not common, then those people can just use docker directly.

>It'd be good to be able to use this package for WordPress Core

That is a great example of something that could be done with an override, but that I think the tool should support out of the box by inspecting if the folder it's running in is not a plugin, but WordPress itself.

Reply:
That's a fair point on `docker-composer.override.yml`. Let's wait and see if there's a need for it.

Reply:
Sounds good.